index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Personal-Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="my-logo">
+        <nav class="menu-bar">
+          <nav class="logo">Richard Obaseki</nav>
+          <div>
+            <i class="fas fa-bars"></i>
+          </div>
```

Comment:
- [ ] Would you please wrap your hamburger button with a`nav` tag.  This is important for semantic and accessibility reasons, especially for people using screen readers. You can visit this [link](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML) if you want to get a deeper understanding.

Reply:
Implemented 👍